### ISSUE TITLE
Assert equals

### ISSUE THREAD

**Issue by LucasZibell** [alu654/tps-Parcial#7]
En vez de comparar todo el tiempo usando assertThat, pueden usar assertEquals que recibe como primer parámetro el valor esperado y como segundo lo que iría antes del == en el assertThat.

**Comment by tostrovsky** [comment]
Hola, cuando intentamos hacer los test, nos da error en los test que estaban relacionados con calcularEnergia, que ustedes bien nos dijeron que lo corrijamos y omitamos los parámetros que no usa. El mensaje del test dice "burns no entiende calcularEnergia(param1, param2)"

**Comment by tostrovsky** [comment]
actualizo: estos errores son los únicos que tiran los test, y ocurre tanto cuando omitimos un parámetro o los dos.

**Comment by LucasZibell** [comment]
mmmm Puede ser que wollok lo tome como varios metodos distintos jajajajaja dependiendo de la cantidad de parametros.
Tira error cuando usaron la variable anonima? `(_, riqueza)`

**Comment by tostrovsky** [comment]
no, pero porque toma "_" como un nombre de un parametro, es como un identificador de un solo caracter ´_´. Por eso, si ponemos calcular( _ , _) toma como que le estamos poniendo el mismo nombre a ambos parámetros también, y rompe

**Comment by LucasZibell** [comment]
Y esto: `(_, __)` 🤔

**Comment by tostrovsky** [comment]
sí,  eso sirve porque toma _ y __ como dos identificadores diferentes. Ahora no rompe nada. Qué pasó? por qué no sirvió lo que nos corrigieron?

**Comment by LucasZibell** [comment]
Digamos que wollok es mistico
